queries.sql

```diff
@@ -79,47 +79,87 @@
 -- SELECT name, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY name;
 
```

Comment:
- [ ] Please remove unused code so that your code base is clean and uncluttered.